### PR TITLE
Updated template due to URP RendererData changes

### PR DESCRIPTION
**Pull request by Verasl**
### Purpose of this PR
Template update for the fixes in this PR:https://github.com/Unity-Technologies/Graphics/pull/5410 it also resolves [this bug](https://fogbugz.unity3d.com/f/cases/1357377/)
Some additional files had general updated to serialisation from current URP, I have added these also as they are low risk and will be experienced by the user regardless when opening the template.

---
### Testing status
Tested manually by QA

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.template-universal/ProjectSettings/QualitySettings.asset [hunk 1]
```diff
@@ -93,7 +93,7 @@ QualitySettings:
     skinWeights: 255
     textureQuality: 0
     anisotropicTextures: 1
-    antiAliasing: 0
+    antiAliasing: 2
```
Comment (phi-lira) [inline com.unity.template-universal/ProjectSettings/QualitySettings.asset:96]: Is this change intentional?

**Review by phi-lira — APPROVED** [review]

File: com.unity.template-universal/ProjectSettings/QualitySettings.asset (hunk 1 above)
Comment (Verasl) [reply com.unity.template-universal/ProjectSettings/QualitySettings.asset:96]: my guess is yes, I didn't change any AA settings but my assumption here is that the Template hasn't had this file updated since URP started to set this depending on it's state, we still haven't separated the logic here right?
